src/main/java/seedu/address/model/person/Person.java

```diff
@@ -102,6 +110,50 @@ public Address getAddress() {
         return address.get();
     }
 
+    public void setPostalCode(PostalCode postalCode) {
```

Comment:
Since this method is not trivial, requiring `postalCode` to be non null, header comments should be added